app/assets/stylesheets/application.scss

```diff
@@ -26,15 +34,73 @@ body {
   text-align: center;
 }
 
-.welcome {
-  background: rgba(255, 255, 255, 0.5);
-  padding: 10px;
-}
-
-.footer {
-  background-color: #333333;
+footer {
   bottom: 0;
+  background-color: $darkslategrey;
   color: #ffffff;
-  position: fixed;
-  width: 100%;
+}
+
+.top-bar ul li {
+  background: $darkslategrey;
+}
+
+.top-bar a {
+  font-size: 16px;
+}
+
+.top-bar-section img {
+  width: 3.5em;
+  max-height: 3.5em;
+  float: right;
+  margin-top: 7px;
+  margin-right: 0px;
+  background: inherit;
+}
+// Footer
+html, body {
+	height: 100%;
+}
+
+.page-wrap {
+  min-height: 100%;
+  /* equal to footer height */
+	margin-bottom: -142px;
+}
+
+.page-wrap:after {
+  content: "";
+  display: block;
+}
+
+.site-footer, .page-wrap:after {
+  /* .push must be the same height as footer */
+	height: 142px;
+}
+
+.site-footer {
+  background: $darkslategrey;
+}
+
+#icon {
+  width: 100px;
+}
+
+.large-4 {
+  background:rgba(255,255,255,0.5);
+  border: 5px solid $mediumaquamarine;
+}
+
+.panel {
+  margin-top: 5px;
```

Comment:
Properties should be ordered border-color, border-left-width, border-right-width, margin-top